app/src/pages/inside/stepPage/stepGrid/defectType/defectType.jsx

```diff
@@ -68,36 +70,55 @@ PALabel.propTypes = {
   patternTemplates: PropTypes.array.isRequired,
 };
 
-export const DefectType = ({ issue, onEdit, onRemove, patternTemplates }) => (
-  <div className={cx('defect-type')}>
-    <div className={cx('defect-type-labels')}>
-      {issue.ignoreAnalyzer && <IgnoredInAALabel />}
-      {issue.autoAnalyzed && <AALabel />}
-      {!!patternTemplates.length && <PALabel patternTemplates={patternTemplates} />}
-      {issue.issueType && <DefectTypeItem type={issue.issueType} onClick={onEdit} />}
-      <div className={cx('edit-icon')} onClick={onEdit}>
-        {Parser(PencilIcon)}
+export const DefectType = ({ issue, onEdit, onRemove, patternTemplates, events }) => {
+  const { trackEvent } = useTracking();
+  const eventData = issue.issueType.startsWith(TO_INVESTIGATE_LOCATOR_PREFIX);
+  const onClickEdit = (event, cb) => {
```

Comment:
I don't think you need to specify the `cb` function as a parameter here.
You already have access to it here. Just call `onEdit()`.